infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/bootstrapper/KubernetesBootstrapper.java

```diff
@@ -128,14 +129,44 @@ private void injectBootstrapper() throws InfrastructureException {
     kubernetesMachine.exec("chmod", "+x", BOOTSTRAPPER_DIR + BOOTSTRAPPER_FILE);
 
     LOG.debug("Bootstrapping {}:{}. Creating config file", runtimeIdentity, machineName);
-    kubernetesMachine.exec(
-        "sh",
-        "-c",
-        "cat > "
-            + BOOTSTRAPPER_DIR
-            + CONFIG_FILE
-            + " << 'EOF'\n"
-            + GSON.toJson(installers)
-            + "\nEOF");
+
+    List<String> filesToContatenate = new ArrayList<String>();
+    filesToContatenate.add("[");
+    boolean firstOne = true;
+    for (Installer installer : installers) {
+      if (firstOne) {
+        firstOne = false;
+      } else {
+        filesToContatenate.add(",");
+      }
+      filesToContatenate.add(GSON.toJson(installer));
+    }
+    filesToContatenate.add("]");
+    int counter = 1;
+    for (String fileToConcatenate : filesToContatenate) {
+      kubernetesMachine.exec(
+          "sh",
+          "-c",
+          "cat > "
```

Comment:
Why do you need created n files? It'd better use `>>` instead of `>` and cat all content into one file part by part.

Reply:
I'll do the change

Reply:
Changed in commit https://github.com/eclipse/che/pull/8805/commits/1fe3e1403394efecc391f7cbaeb749fc8467e5a8